### ISSUE TITLE
Change "Copyright" footer

### ISSUE THREAD

**Issue by tomschenkjr** [Chicago/open-data-etl-utility-kit#15]
Need to change it from "Tom Schenk and Jonathan Levy" to "City of Chicago and contributors"

**Comment by rgroves** [comment]
It looks like this is controlled by the copyright variable in the file *open-data-etl-utility-kit/docs/conf.py*.

Currently it is set to '2014, City of Chicago'. Is it correct that this would just need to be updated to 'City of Chicago and contributors' (maybe with the current year prepended) and then one would need to perform a Sphinx rebuild of the docs?

**Comment by tomschenkjr** [comment]
Yeah, that's right. It's a fairly easy fix but keeps getting placed at the bottom of the to-do list.

**Comment by rgroves** [comment]
I will attempt to pick this one up. Used Sphinx once about 4 years ago so will need to get it setup to do the doc gen. Sounds like this one shouldn't be too difficult.

**Comment by rgroves** [comment]
I cloned the repository locally, updated conf.py and rebuilt the html docs, but I noticed the change set had more changes than I expected it to. It looks like the files in the docs directory have been modified since the last time the Sphinx built html documentation was committed.

Would you prefer to do an interim commit of the rebuilt documentation based on the docs current state. Then I'll update the copyright info, rebuild and check in that as a separate commit to keep the commits relevant to the changes that were made in them?

**Comment by tomschenkjr** [comment]
Hi @rgroves -- sorry for completely borking this thread!

To the extent you have time to be engaged: that suggestion makes sense. I've been reconciling a few things on this project and then that tees everything up nicely for the next "release".
